test/unit/collection.test.js

```diff
@@ -253,6 +249,8 @@ describe('Collection', function () {
             delete collectionJSON.info.version;
             delete collectionDefinition.info.version;
 
+            collectionJSON.info.description = collectionJSON.info.description.toJSON();
```

Comment:
Add a `TODO` comment to remove this hack after the description serialization bug has been fixed.